### ISSUE TITLE
Feature Request: Seperate nspanel_esphome_advanced.yaml

### ISSUE THREAD

**Issue by fordcrews** [Blackymas/NSPanel_HA_Blueprint#985]
How about keeping an nspanel_esphome_advanced.yaml updated in the github, so those of us with existing ota passwords can simply add ota_password:, web_password:, api_password to our secrets file instead of having to manually download the raw files save them locally, edit them, every time there is a new release.   

It also would  be good to add lambda code to esphome onboot to change any of these if new_ota_password, new_web_password, new_api_password is set in secrets. 

Would also be good to allow you to change the username from admin in secrets file.

**Comment by pbranly** [comment]
I fully agree. 
The issue is to define the minimal configuration that should be in the secret file

**Comment by edwardtfn** [comment]
We are working on making the ESPHome code more customizable and to add this documentation with the next release: https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/dev/docs/en/customization.md

Some of those will be possible with v3.4, but some will have to wait for the next version to be available (as we added some missing id's to components).

Please take a look and let me know your thoughts.

**Comment by adamengineering** [comment]
When validating the Installation file to check if the customization is OK, i get the following error:

`networks: 
    - 
      Source for extension of ID 'wifi_default' was not found.
      id: !extend wifi_default`

Any suggestion how to fix this error is welcome ;-)

**Comment by edwardtfn** [comment]
`wifi_default` will be introduced with the next release (or the code in `dev` branch). It won't work with v3.4.

**Comment by fordcrews** [comment]
> We are working on making the ESPHome code more customizable and to add this documentation with the next release: https://github.com/Blackymas/NSPanel_HA_Blueprint/blob/dev/docs/en/customization.md
> 
> Some of those will be possible with v3.4, but some will have to wait for the next version to be available (as we added some missing id's to components).
> 
> Please take a look and let me know your thoughts.

What is the opposite of Logging Verbose, I noticed when I cut logging off  flashing the nextion seem to always be successful.  

All the rest seem to cover all my issues, except an easy way to change the OTA password from one thing to another.

**Comment by edwardtfn** [comment]
> What is the opposite of Logging Verbose, I noticed when I cut logging off flashing the nextion seem to always be successful.

Do you mean it is failing when you have verbose logging?

If you don't set any logger level, it will be `DEBUG` by default.
Those are the possible levels: https://esphome.io/components/logger.html#log-levels

**Comment by fordcrews** [comment]
OOps was out of town.  If I bring the nspanel_esphome.yaml local and remove logger: section completely, or set it to none, tft flashing seems to work 100% of the time.   Otherwise, I seem to fail loading the display.   I've flashed 15 of them, first 3 took me forever to get nspanel_us.tft to upload without timing out.   Last 12, I turned off logging and they flashed without any trouble at all.     Maybe something else changed, or my network/home assist is odd, but that  is what I observed.
